other/test-setup/setup-test-env.ts

```diff
@@ -1,22 +1,12 @@
 import './setup-env-vars.ts'
+import { afterAll, afterEach } from 'vitest'
 import { installGlobals } from '@remix-run/node'
-import { matchers, type TestingLibraryMatchers } from './matchers.cjs'
 import 'dotenv/config'
 import fs from 'fs'
 import { BASE_DATABASE_PATH, DATABASE_PATH } from './paths.ts'
 import { deleteAllData } from './utils.ts'
 import { prisma } from '~/utils/db.server.ts'
 
-declare global {
-	namespace Vi {
-		interface JestAssertion<T = any>
-			extends jest.Matchers<void, T>,
-				TestingLibraryMatchers<T, void> {}
-	}
-}
-
-expect.extend(matchers)
```

Comment:
I think we do still need to extend expect somehow so these matchers are available at runtime.

Reply:
![vitest](https://github.com/epicweb-dev/epic-stack/assets/3650909/6c41f3df-c1ea-44c9-8478-ae7e9a4d5eb9)

https://vitest.dev/api/expect.html#expect-extend

Reply:
Yes

we need `expect.extend(matchers)` back